### ISSUE TITLE
New install, no pf database?

### ISSUE THREAD

**Issue by loganess** [inverse-inc/packetfence#8041]
**Describe the bug**
Hello, 

I did fresh install using both ways: ISO and install on existing linux debian.
There is no "pf" database after finished the installation.

Issue is happening with with 13.1 and Devel

from the packetfence.log

```
ERROR: unable to connect to database: Access denied for user 'pf'@'localhost' (pf::db::db_connect)
```

**To Reproduce**
Fresh install using ISO or existing debian linux 11 amd64

**Screenshots**

**Comment by loganess** [comment]
i reinstalled 13.1 on both redhat and rocky, it works fine, i can access the management url and no more error (before i did same but it didn't work)

**Comment by JeGoi** [comment]
Hi there,
Devel is unstable, so I am not surprised.
Until now, I am not able to reproduce. No issues with debian iso for me.
Questions:
- Which hypervisor?
- Did you use default values in the wizard or create specifics?

(I will try to reproduce)

**Comment by JeGoi** [comment]
Could it be related to that: https://github.com/inverse-inc/packetfence/issues/8144 ?

**Comment by stgmsa** [comment]
Hi @loganess , @JeGoi 

I think it's probably the same issue we met in #8144 after checking the backend logs.
Typically, any API call that reads/writes the database will cause a 500 error and "unknown error, check logs" in the admin UI.
I got a temporary fix: manually restart `pfperl-api` service using `systemctl restart packetfence-pfperl-api` after went through the configurator that fixes the issue of #8144, Hopefully it works for this issue too.

we'll go through the issues and get this fixed in a +1 patch.

**Comment by JeGoi** [comment]
Fixed in 14.0
